.github/workflows/build-and-test.yml

```diff
@@ -1,7 +1,7 @@
 name: Build and Test
 
 env:
-  MG_VERSION: "2.1.1"
+  MG_VERSION: "2.3.0"
```

Comment:
Why are timeot error changes shown in Files changed here? They are already in develop